packages/angstrom-async/angstrom-async.0.12.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "angstrom-async"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/angstrom"
+bug-reports: "https://github.com/inhabitedtype/angstrom/issues"
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {>= "1.0"}
+  "angstrom" {>= "0.7.0"}
```

Comment:
```suggestion
  "angstrom" {= version}
```
Are you sure it works with such an old version? I'd more recommend to stick with the same version for the same "set of packages".